markets/hydration/reservesConfigs.ts

```diff
@@ -66,8 +66,8 @@ export const strategyDOT: IReserveParams = {
   reserveDecimals: "10",
   aTokenImpl: eContractid.AToken,
   reserveFactor,
-  supplyCap: "5000000",
-  borrowCap: "1000000",
+  supplyCap: "22,222,222".replaceAll(",", ""),
+  borrowCap: "500000",
```

Comment:
why do you change borrow cap?

Reply:
changes like this should be done separatelly from PR

Reply:
repo is out of sync with chain state. We have only task that reviews and updates everything

Reply:
just push changes like that to master directly, as they are not related to the proposal